server/endpoints.ts

```diff
@@ -1,10 +1,42 @@
 import express from "express";
 import { validationResult, ValidationChain } from "express-validator";
-import { totalReviews, howManyReviewsEachClass, howManyEachClass, topSubjects, getReviewsOverTimeTop15 } from "./endpoints/AdminChart";
-import { getReviewsByCourseId, getCourseById, insertReview, insertUser, getCourseByInfo, incrementLike, decrementLike } from "./endpoints/Review";
+import {
+  totalReviews,
+  howManyReviewsEachClass,
+  howManyEachClass,
+  topSubjects,
+  getReviewsOverTimeTop15,
+} from "./endpoints/AdminChart";
+import {
+  getReviewsByCourseId,
+  getCourseById,
+  insertReview,
+  insertUser,
+  getCourseByInfo,
+  incrementLike,
+  decrementLike,
+} from "./endpoints/Review";
+import {
+  countReviewsByStudentId,
+  getTotalLikesByStudentId,
+  getReviewsByStudentId,
+} from "./endpoints/Profile";
```

Comment:
looks like you need to set your merge base branch to `my_reviews_page` to get access to the profile endpoints